docs/deploy_pg_analytics.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Deploying `pg_analytics`
+---
+
+This section will guide you on how to deploy `pg_analytics` as an extension to an existing PostgreSQL database. You must have superuser access to the database to install the extension.
+
+Note: We only provide instructions for deploying on Linux Debian/Ubuntu. If you are using a different version of Linux, please contact [us](mailto:sales@paradedb.com) and we'll be happy to help.
+
+# Step-by-Step Guide
+
+## Installing `pg_analytics`
+
+ParadeDB provides pre-built binaries for the `pg_analytics` extension for Postgres 15 and 16, for Ubuntu 22.04. You can find them in the [GitHub Releases](https://github.com/paradedb/paradedb/releases/latest). If you
+are using a different version of Postgres or a different operating system, you will need to build the extension from source.
+
+### Using pre-build binaries
+
+```bash
+curl -L "https://github.com/paradedb/paradedb/releases/download/v0.6.1/pg_analytics-v0.6.1-pg16-arm64-ubuntu2204.deb" -o /tmp/pg_analytics.deb
+sudo apt-get install -y /tmp/*.deb
+```
+
+Note: You can replace `v0.6.1` with the `pg_analytics` version you wish to install, and `pg16` with the version of Postgres you are using.
+
+### Building from source
+
+You can find the instructions for doing so [here](https://github.com/paradedb/paradedb/tree/v0.6.1/pg_analytics#installation).
+
+## Updating `postgresql.conf`
+
+To enable `pg_analytics`, you will need to add `pg_analytics` to your `shared_preload_libraries` in your `postgresql.conf` file:
+
+```conf
+shared_preload_libraries = 'pg_analytics'
+```
+
+You will then need to reload your PostgreSQL server for the changes to take effect.
+
+## Loading the Extension
+
+Once the extension binary is installed on your system, connect to your Postgres database via your client of choice (e.g. `psql`) and run the following command:
+
+```sql
+CREATE EXTENSION pg_analytics CASCADE;
+```
```

Comment:
I think the `CASCADE` implies that other extensions also need to be installed? Is that `pg_search`? It would be worth explaining what else is getting installed, and whether they also need to be in `shared_preload_libraries`.

Reply:
Yeah I can remove the CASCADE, it's not necessary